Jenkinsfile

```diff
@@ -92,9 +92,6 @@ pipeline {
       }
     }
     stage("Push Ingress Image") {
-      when {
-        anyOf { branch 'controller-fips'; buildingTag() }
-      }
```

Comment:
Maybe put the condition around `sh 'echo ${REGISTRY}/${IMAGE_NAME}:${TAG} >> images.list'` so that it does everything but the finalizeBuild step with the new image.